datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -204,6 +204,19 @@ def __init__(self, *args, **kwargs):
         self.disable_generic_tags = (
             is_affirmative(self.instance.get('disable_generic_tags', False)) if instance else False
         )
+
+        self.global_metrics_filter = self.instance.get('global_metrics_filter') if instance else None
```

Comment:
Do you think there is a use case for configuring `global_metrics_filter` as a list of strings? It'll make it easier for users to configure, but a single string works too.